src/electrum/server.rs

```diff
@@ -601,6 +569,55 @@ impl Connection {
         }
     }
 
+    fn handle_value(
+        &mut self,
+        cmd: Value,
+        empty_params: &Value,
+        start_time: Instant,
```

Comment:
Shouldn't the start_time start ticking separately for each `handle_value` call?

Reply:
yes, it makes sense. Done in b8f4524cdbe8f280da9fd87d2a4ac136f630a30f